autogpt_platform/frontend/src/components/molecules/Dialog/components/BaseFooter.tsx

```diff
@@ -17,15 +17,15 @@ export function BaseFooter({
 
   return ctx.isLargeScreen ? (
     <div
-      className={`flex justify-end gap-4 pt-6 ${className}`}
+      className={`sticky bottom-0 mt-auto flex justify-end gap-4 border-t border-neutral-200 bg-white pt-6 dark:border-neutral-800 dark:bg-neutral-950 ${className}`}
```

Comment:
This could break the layout on some other pages using the modal, have you checked the usage there is fine after these changes?